free-programming-books-ru.md

```diff
@@ -235,6 +236,7 @@
 * [Руководство по Java Core](http://proselyte.net/tutorials/java-core) - Евгений Сулейманов
 * [Руководство по Servlets](http://proselyte.net/tutorials/servlets) - Евгений Сулейманов
 * [Руководство по языку программирования Java](https://metanit.com/java/tutorial) - Евгений Попов
+* [Самоучитель по Java в игровой форме](https://javarush.ru)
```

Comment:
not a book. Maybe the Russsian page has been lax about separating genres. What do you think?

Reply:
Yes, you are right, it's not a book. It's my fail, i will fix it.